utils_go17.go

```diff
@@ -0,0 +1,40 @@
+// +build go1.7
+// +build !go1.8
+
+// Go MySQL Driver - A MySQL-Driver for Go's database/sql package
```

Comment:
Please move them to the top, i.e. above the build tags (like https://github.com/go-sql-driver/mysql/pull/603/files)